### ISSUE TITLE
kcl-bootstrap can't download dependencies on Maven repository

### ISSUE THREAD

**Issue by nezetic** [awslabs/amazon-kinesis-client-nodejs#76]
The _kcl-bootstrap_ script can't download dependencies on Maven repository anymore.

The root cause is pretty straight forward (and frightening): the script does **not** respect the URL scheme, and **never** use _https_ while downloading dependencies.

The issue raised because Maven repository does not accept _http_ requests anymore.

I submitted a PR solving this #75

**Comment by obuley** [comment]
Any ideas on quick local fix?

**Comment by rjmohammadww** [comment]
any status update?

**Comment by HansonVPHAM** [comment]
I got a 501 error code when downloading... 
We must use HTTPS instead of HTTP.
Looking the way to fix that issue...
Please, anyone have workarounds?

**Comment by rossmckelvie** [comment]
here is a workaround we are using. Replace your line in package.json with this:

```
"aws-kcl": "git://github.com/awslabs/amazon-kinesis-client-nodejs.git#4adcebeb3f110494af1b72e739e0916ddaeda6c9",
```

**Comment by jushkem** [comment]
Released as part of 2.1.0, closing this out.
